### ISSUE TITLE
Not a bug but a feature request

### ISSUE THREAD

**Issue by fred-sheehan** [realmazharhussain/gdm-settings#124]
### What happened?

I see you have a sound section to change sound defaults on the GDM log in screen, and I know many friends who use screen readers, which of course are easily configurable in gnome, and work well, but don't however work on the log in screen, and so I was wondering if this could be added as a welcome extra tweak?

### Expected behavior

Have a log in screen that would have the screen reader on and talk to the user who wants to log on

### To Reproduce

Any gnome log in screen doesn't have screen reader enabled amking log in screen combersome for partially sighted users.

### Application Version

Latest

### Source

Github

### Environment

- **OS**: 
- **GNOME**:

**Comment by realmazharhussain** [comment]
I'm pretty sure GDM already has a Screen Reader option in its accessibility menu.

**Comment by fred-sheehan** [comment]
GDM does, however, the default GUI install is XFCE which doesn't read anything to you at install, and if you try and use the 'speech assisted'install, you don't get a desktop, you just end up at a plain command prompt for a 'minimnal install' even though nyou might have asked for several desktops.

In the GUI install, it asks which DE you want to use, and then configures that for you. The speech assisted install takes no such steps, it doesn't even ask for a preference,  and so you get no login screen or DE working at all upon a reboot 'to your new install'

**Comment by fred-sheehan** [comment]
Have you ever tried a 'speech assisted' install?

**Comment by realmazharhussain** [comment]
> GDM does, however, the default GUI install is XFCE which doesn't read anything to you at install, and if you try and use the 'speech assisted'install, you don't get a desktop, you just end up at a plain command prompt for a 'minimnal install' even though nyou might have asked for several desktops.
> 
> In the GUI install, it asks which DE you want to use, and then configures that for you. The speech assisted install takes no such steps, it doesn't even ask for a preference, and so you get no login screen or DE working at all upon a reboot 'to your new install'

What are you talking about?

I cannot change your distributions' install process.
